src/media/css/app-list.styl

```diff
@@ -0,0 +1,72 @@
+/**
+ * App listing styles.
+ */
+
+@import 'lib';
+
+.app-list {
+    position: absolute;
+    top: $app-preview-height;
+    bottom: 0;
+    left: 0;
+
+    width: 100%;
+    overflow: auto;
+
+    ul {
+        padding: 4em 3em 3em 4em;
+    }
+
+    li {
+        float: left;
+        margin: 0 1em 1em 0;
+
+        list-style: none;
+    }
+}
+
+.app-list-app {
+    position: relative;
+
+    width: 150px;
+    height: 150px;
+    outline: 0;
+    border-radius: 100%;
+    overflow: hidden;
+
+    color: $greyscale-light-grey;
+    background-color: $greyscale-light-grey;
+
+    backface-visibility: hidden;
+
+    transition: border-width .5s,
+                transform .5s;
+
+    &:focus {
+        z-index: 1;
+
+        border: 8px solid $greyscale-white;
+
+        transform: scale(1.5);
+    }
+
+    img {
+        width: 100%;
+        height: auto;
+    }
+
+    .name {
+        position: absolute;
+        bottom: 0;
+        left: 0;
```

Comment:
I guess the answer is yes. To be honest, I've never deal with RTL languages before. However in this case, I think it should make no difference for RTL languages since the text is centered, right?
```
width: 100%;
text-align: center;
```